(maint) Message and string fixes

DavidS opened the pull request.

jbondpdx commented on locales/pdk.pot:
```diff
@@ -227,26 +229,46 @@ msgstr ""
 
-#: ../lib/pdk/cli/exec.rb:155
+#: ../lib/pdk/cli/exec.rb:126
+msgid "PUPPET_GEM_VERSION is not supported by PDK. Please use the --puppet-version option on your PDK command or set the PDK_PUPPET_VERSION environment variable instead"
```
Drop the "Please", please. :) And end the sentence with a period.

jbondpdx commented on locales/pdk.pot:
```diff
@@ -814,10 +899,22 @@ msgstr ""
 
-#: ../lib/pdk/module/build.rb:149
+#: ../lib/pdk/module/build.rb:127
+msgid "%{message} Please rename the file or exclude it from the package by adding it to the .pdkignore file in your module."
```
Remove the "Please".

DavidS commented:
@jbondpdx no worries, I've got so many plates spinning, one more or less doesn't make a dent.

Updated the messages and rebased the PR to current master.

glennsarti commented:
@DavidS Looks like this needs a rebase on master :-(

DavidS commented:
Done.
